### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Dadadon**
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- Mobile popup
- Desktop popup

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -461,5 +259,7 @@ <h2 class="contact-text">Contact me</h2>
         </div>
       </footer>
     </section>
+
+    <script src="popup.js"></script>
```
Comment (SajjadAhmad14) [inline index.html:263]: The pop-up image should be the full width of the container on desktop view. Moreover, the technologies list and buttons should be on the right side of the container, and the text should be on the left side. Please follow the design for the desktop view. Kindly see the screenshot below and fix the design issues.
desktop

File: index.html [hunk 2]
```diff
@@ -98,231 +93,34 @@ <h1 class="suptext">LET'S CONNECT</h1>
...
+            </div>
+          </div>
         </div>
-      </section>
+      </div>
     </div>
+    <ul class="card-list" id="cards"></ul>
     <div id="about" class="grid-container2">
       <section class="card3">
         <h1 class="a-head">About<br />Myself</h1>
```
Comment (SajjadAhmad14) [inline index.html:126]: Please add linter config for html+css+js from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link.

File: style.css [hunk 3]
```diff
@@ -4,14 +4,15 @@ html * {
```
Comment (SajjadAhmad14) [inline style.css:4]: [ OPTIONAL ] Please use a directory for all of the images in the project directory. This way project directory looks clean and managed.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Davian Bertoni, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (SajjadAhmad14) [inline eslintrc.json:1]: Please add linter config for html+css+js from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link. Currently, you have not configured linter for JavaScript.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Davian Bertoni, 
 ### Thank you for implementing changes suggested previously :ok_hand:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: eslintrc.json (hunk 4 above)
Comment (Dadadon) [reply eslintrc.json:1]: I dont understand. I followed all the steps provided in the link

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Davian Beroni, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**